<!DOCTYPE html>

<html lang="en">

<head>
    <meta charset="utf-8">
    <title>Publications</title>
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
    <link href="css/styles.css" type="text/css" rel="stylesheet" />
    <link href="pictures/favicon.png" type="image/png" rel="shortcut icon"> <!-- Favicon -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
</head>

<body class="body-publications">
    
    <header>
        <!-- Navigation bar + logo -->

        <!-- RESPONSIVE AND TOGGLEABLE NAVBAR -->
        <nav class="navbar fixed-top navbar-dark bg-transparent navbar-expand-sm bg-company-black">
            <!-- bg-company-black css-->
            <div class="container-fluid">
                <a href="/">
                    <img class="navbar-brand navbar-dark py-0 mt-3 ml-15" id="logo" src="icons/logo.png" alt="logo" title="Logo">
                </a>
                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#myNavbar"><span class="navbar-toggler-icon"></span></button>
                <div class="collapse navbar-collapse" id="myNavbar">
                    <ul class="navbar-nav mx-auto">
                        <li class="nav-item  py-0 ml-4"><a class="nav-link text-white" href="index.html">
                                HOME
                            </a></li>
                        <li class="nav-item  py-0 ml-4"><a class="nav-link text-white" href="projects.html">
                                PROJECTS
                            </a></li>
                        <li class="nav-item  py-0 ml-4"><a class="nav-link text-white" href="aboutus.html">
                                ABOUT US
                            </a></li>
                        <li class="nav-item   py-0 ml-4"><a class="nav-link text-white" href="ourteam.html">
                                OUR TEAM
                            </a></li>
                        <li class="nav-item   py-0 ml-4 active"><a class="nav-link" href="publications.html">
                                PUBLICATIONS
                            </a></li>
                    </ul>
                </div>
            </div>
        </nav>
    </header>
    
    <div id="banner-publications"></div>
        <div class="row justify-content-center align-items-center title">
            <h1 class="text-white d-none d-block aTitle"> PUBLICATIONS </h1>
        </div>
    <div class="pusher d-flex justify-content-center align-items-end"><a href="#arrow"><img class="" id="arrow" src="icons/down-arrow.png" alt="down" title="down"></a></div>
    <!----------------------PUBLICATIONS CONTENT------------------------------->
    <div class="container-fluid services">
        
        <!---------------------PUBLICATIONS BANNER------------------------>
        <div class="row py-5 banner">
            <div class="d-flex col-md-4 col-sm-4 justify-content-center">
                <figure>
                    <img src="pictures/underconstruction.png" alt="java" title="Under Construction...">
                    <figcaption class="mt-3 text-white position-absolute">Under Construction...</figcaption>
                </figure>
            </div>
            <div class="col-md-4 col-sm-4 justify-content-center d-flex">
                <figure>
                    <img src="pictures/underconstruction.png" alt="css3" title="Under Construction...">
                    <figcaption class="mt-3 text-white position-absolute">Under Construction...</figcaption>
                </figure>
            </div>

            <div class="col-md-4 col-sm-4 justify-content-center d-flex">
                <figure>
                    <img src="pictures/underconstruction.png" alt="html5" title="Under Construction...">
                    <figcaption class="mt-3 text-white position-absolute">Under Construction...</figcaption>
                </figure>
            </div>
            
            </div>
            
            
            
            <!-------------PUBLICATIONS INFO------------------>
            <div id="infobanner" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Cankara, F., & Doğan, T. (2023). ASCARIS: Positional feature annotation and protein structure-based representation of single amino acid variations. Computational and Structural Biotechnology Journal.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Lobentanzer, S., Aloy, P., Baumbach, J., Bohar, B., Carey, V. J., Charoentong, P., ... & Saez-Rodriguez, J. (2023). Democratizing knowledge representation with BioCypher. Nature Biotechnology, 1-4.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Dalkıran, A., Atakan, A., Rifaioğlu, A. S., Martin, M. J., Atalay, R. Ç., Acar, A. C., ... & Atalay, V. (2023). Transfer learning for drug–target interaction prediction. Bioinformatics, 39(Supplement_1), i103-i110.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Yüksel, A., Ulusoy, E., Ünlü, A., & Doğan, T. (2023). Selformer: Molecular representation learning via selfies language models. Machine Learning: Science and Technology.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Target Specific De Novo Design of Drug Candidate Molecules with Graph Transformer-based Generative Adversarial Networks
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Atas Guvenilir, H., & Doğan, T. (2023). How to approach machine learning-based prediction of drug/compound–target interactions. Journal of Cheminformatics, 15(1), 1-36.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Özdilek, A. S., Atakan, A., Özsarı, G., Acar, A., Atalay, M. V., Doğan, T., & Rifaioğlu, A. S. (2023). ProFAB—open protein functional annotation benchmark. Briefings in bioinformatics, 24(2), bbac627.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        UniProt: the universal protein knowledgebase in 2023. Nucleic Acids Research, 2023, 51.D1: D523-D531.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Ciray, F., & Doğan, T. (2022). Machine Learning-based Prediction of Drug Approvals Using Molecular, Physicochemical, Clinical Trial and Patent Related Features. Expert Opinion on Drug Discovery [doi:10.1080/17460441.2023.2153830].
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Unsal, S., Atas, H., Albayrak, M., Turhan, K., Acar, A. C., & Doğan, T. (2022). Learning Functional Properties of Proteins with Language Models. Nature Machine Intelligence, 4(3), 227-245
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded mb-3">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Özsarı, G., Rifaioglu, A. S., Atakan, A., Doğan, T., Martin, M. J., Atalay, R. C., & Atalay, V. (2022). SLPred: A Multi-view Subcellular Localization Prediction Tool for Multi-location Human Proteins. Bioinformatics, 38(17), 4226–4229
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        UniProt Consortium (2022). UniProt: the Universal Protein Knowledgebase in 2023. Nucleic Acids Research, gkac1052 [doi:10.1093/nar/gkac1052].
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Doğan, T., Güzelcan, E. A., Baumann, M., Koyas, A., Atas, H., Baxendale, I., Martin, M., & Cetin-Atalay, R. (2021). Protein Domain-Based Prediction of Compound–Target Interactions and Experimental Validation on LM Kinases. PLOS Computational Biology, 17(11), e1009171.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Yakimovich, A., Özgür, A., Doğan, T., Ozkirimli, E. (2021). Machine Learning Methodologies to Study Molecular Interactions. Frontiers in Molecular Biosciences, 8, 1174.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Cetin-Atalay, R., Kahraman, D. C., Nalbat, E., Rifaioglu, A. S., Atakan, A., Donmez, A., Atas, H., Atalay, M.V., Acar, A.C., Doğan, T., (2021). Data Centric Molecular Analysis and Evaluation of Hepatocellular Carcinoma Therapeutics Using Machine Intelligence-Based Tools. Journal of Gastrointestinal Cancer, 1-11.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Doğan, T., Atas, H., Joshi, V., Atakan, A., Rifaioglu, A.S., Nalbat, E., Nightingale, A., Saidi, R., Volynkin, V., Zellner, H. Cetin-Atalay, R., Martin, M. J. & Atalay, V. (2021). CROssBAR: Comprehensive Resource of Biomedical Relations with Knowledge Graph Representations. Nucleic Acids Research, 49(16), e96-e96.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        UniProt Consortium. (2021). UniProt: the universal protein knowledgebase in 2021. Nucleic Acids Research, 49(D1), D480-D489.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Wang, Y., Wang, Q., Huang, H., Huang, W., Chen, Y., McGarvey, P. B., ... & UniProt Consortium. (2021). A crowdsourcing open platform for literature curation in UniProt. PLoS biology, 19(12), e3001464.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Cichońska, A., Ravikumar, B., Allaway, R.J., …, The IDG-DREAM Drug-Kinase Binding Prediction Challenge Consortium, et al. (2021). Crowdsourced mapping of unexplored target space of kinase inhibitors. Nature Communications, 12, 3307 [doi:10.1038/s41467-021-23165-1].
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Rifaioglu, A.S., Nalbat, E., Atalay, M.V., Martin, M.J., Cetin-Atalay, R. & Doğan, T. (2020). DEEPScreen: High Performance Drug-Target Interaction Prediction with Convolutional Neural Networks Using 2-D Structural Compound Representations. Chemical Science, 11(9), 2531-2557.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Rifaioglu, A. S., Cetin Atalay, R., Cansen Kahraman, D., Doğan, T., Martin, M., & Atalay, V. (2020). MDeePred: Novel Multi-Channel protein featurization for deep learning based binding affinity prediction in drug discovery. Bioinformatics, 37(5), 693-704.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Donmez, A., Rifaioglu, A. S., Acar, A., Doğan, T., Cetin-Atalay, R., & Atalay, V. (2020). iBioProVis: Interactive Visualization and Analysis of Compound Bioactivity Space. Bioinformatics, 36(14), 4227-4230.</p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Rifaioglu, A.S., Atas, H., R., Martin, M.J., Cetin‐Atalay, R., Atalay, M.V. & Doğan, T. (2019). Recent applications of deep learning and machine intelligence on in silico drug discovery: methods, tools and databases. Briefings in Bioinformatics, 20(5), 1878-1912.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Rifaioglu, A.S., Doğan, T., Martin, M.J., Cetin-Atalay, R. & Atalay, M.V. (2019). DEEPred: Automated Protein Function Prediction with Multi-task Feed-forward Deep Neural Networks. Scientific reports, 9(1), 7344.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        UniProt Consortium. (2019). UniProt: a worldwide hub of protein knowledge. Nucleic acids research, 47(D1), D506-D515.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Zhou, N., Jiang, Y., Bergquist, T.R., Lee, A.J., Kacsoh, B.Z., Crocker, A.W., Lewis, K.A., Georghiou, G., Nguyen, H.N., Hamid, M.N., Davis, L., Doğan, T., et al. (2019). The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens. Genome Biology, 20, 244 [doi:10.1186/s13059-019-1835-8].
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Garcia, L., Bolleman, J., Gehant, S., Redaschi, N., Martin, M. & UniProt Consortium (2019). FAIR adoption, assessment and challenges at UniProt. Scientific Data, 6(1), 1-4.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Dalkiran, A., Rifaioglu, A.S., Atalay, M.V., Martin, M.J., Cetin‐Atalay, R. & Doğan, T. (2018). ECPred: A Tool for the Prediction of the Enzymatic Functions of Protein Sequences Based on the EC Nomenclature. BMC Bioinformatics, 19(1), 334.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Demirel, H.C., Doğan, T. & Tuncbag, N. (2018). A Structural Perspective on Modulation of Protein-Protein Interactions with Small Molecules. Current Topics in Medicinal Chemistry, 18(8):700-713.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Doğan, T. (2018). HPO2GO: Prediction of Human Phenotype Ontology Term Associations For Proteins Using Cross Ontology Annotation Co-occurrences. PeerJ 6:e5298.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Rifaioglu, A.S., Doğan, T., Saraç, Ö.S., Ersahin, T., Saidi, R., Atalay, M.V., Martin, M.J. & Cetin‐Atalay, R. (2018). Large‐scale automated function prediction of protein sequences and an experimental case study validation on PTEN transcript variants. Proteins: Structure, Function, and Bioinformatics, 86(2), 135-151.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Pichler, K., et al. & UniProt Consortium. (2018). SPIN: Submitting Sequences Determined at Protein Level to UniProt. Current protocols in bioinformatics, e52.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Poux, S., et al. & UniProt Consortium. (2017). On expert curation and scalability: UniProtKB/Swiss-Prot as a case study. Bioinformatics, 33(21), 3454-3460.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Zaru, R., et al. & UniProt Consortium. (2017). From the research laboratory to the database: the CaenorhUSAitis elegans kinome in UniProtKB. Biochemical Journal, 474(4), 493-515.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        UniProt Consortium. (2017). UniProt: the universal protein knowledgebase. Nucleic acids research, 45(D1), D158-D169.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Doğan, T., MacDougall, A., Saidi, R., Poggioli, D., Bateman, A., O’Donovan, C., & Martin, M. J. (2016). UniProt-DAAC: Domain Architecture Alignment and Classification, a New Method for Automatic Functional Annotation in UniProtKB. Bioinformatics, 32(15): 2264-2271
                     </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Dalkiran, A., Rifaioglu, A.S., Atalay, M.V., Martin, M.J., Cetin‐Atalay, R. & Doğan, T. (2018). ECPred: A Tool for the Prediction of the Enzymatic Functions of Protein Sequences Based on the EC Nomenclature. BMC Bioinformatics, 19(1), 334.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Jiang, Y., Oron, T. R., Clark, W. T., Bankapur, A. R., D'Andrea, D., Lepore, R., Funk, C.S. Kahanda, I., Verspoor, K.M., Ben-Hur, A., Koo, E., Penfold-Brown, D., Shasha, D., Youngs, N., Bonneau, R., Lin, A., Sahraeian, S.M, Martelli, P.L., Profiti, G., Casadio, R., Cao, R., Zhong, Z., Cheng, J., Altenhoff, A., Skunca, N., Dessimoz, C., Doğan, T., et al. (2016). An expanded evaluation of protein function prediction methods shows an improvement in accuracy. Genome Biology, 17:184 [doi:10.1186/s13059-016-1037-6].
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Breuza, L., et al. & UniProt Consortium. (2016). The UniProtKB guide to the human proteome. Database, bav120.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        UniProt Consortium. (2015). UniProt: a hub for protein information. Nucleic Acids Research, 43(D1): D204-D212.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Ison, J., Rapacki, K., Ménager, H., Kalaš, M., Rydza, E., Chmura, P., Anthon, C., Beard, N., Berka, K., Bolser, D., Booth, T., Bretaudeau, A., Brezovsky, J., Casadio, R., Cesareni, G., Coppens, F., Cornell, M., Cuccuru, G., Davidsen, K., Della Vedova, G., Doğan, T., et al. (2015). Tools and data services registry: a community effort to document bioinformatics resources. Nucleic acids research, 44(D1), D38-D47.
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        UniProt Consortium. (2014). Activities at the Universal Protein Resource (UniProt). Nucleic Acids Research, 42(D1), D191-D198 [Corrigendum: doi:10.1093/nar/gku469].
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>

            <div id="infobanner2" class="row d-flex justify-content-around text-center">
                
                <div class="col-lg-6 col-md-8 col-sm-10 ol-10 border border-white rounded">
                    <h2 class="text-white">*</h2>
                    <p class="text-white">
                        Doğan, T. and Karaçalı, B. (2013). Automatic Identification of Highly Conserved Family Regions and Relationships in Genome Wide Datasets Including Remote Protein Sequences. PLoS ONE 8(9): e75458 [doi:10.1371/journal.pone.0075458].
                    </p>
                    <h2 class="text-white">*</h2>
                </div>

            </div>
            
        
    </div>
    


    <!-------------------------------FOOTER------------------------------>
    <div class="">
        <footer class="container-fluid w-100 text-light pt-3 footer footerOther">
            <div class="container-fluid pt-3 pl-0 text-center text-md-left footerPart1 mb-3 p-3">
                <div class="row">
                    <div class="col-12 text-center">
                        <h2>LET'S CONNECT!</h2>
                    </div>
                    <div class="col-12 text-center">
                        <p>Academics &amp; Students stay up to date with the latest paper updates &amp; annoucements </p>
                    </div>
                    <div class="col-12 text-center">
                        <form class="connection" action="">
                            <label class="conLabel" for="con">Enter your email adress:</label>
                            <input class="conField" type="text" name="con" id="con">
                            <input class="conButton" type="submit" value="SUBMIT"> </form>
                    </div>
                </div>
            </div>

            <div class="row justify-content-center">
                <div class="col-md-4 mb-md-0 mb-0">
                    <ul class="list-unstyled d-inline d-flex justify-content-center">
                        <li><a href="https://github.com/HUBioDataLab"><img class="img-fluid" src="icons/github-logo.png" alt="github" title="Visit Github"></a></li>
                        <li><a href="https://huggingface.co/HUBioDataLab"><img class="img-fluid" src="icons/huggingface-crop.png" alt="instagram" title="Visit Huggingface Space"></a></li>
                        <li><a href="https://twitter.com/tuncadogan/"><img class="img-fluid" src="icons/twitter.png" alt="twitter" title="Visit Twitter"></a></li>
                        
                    </ul>
                </div>
            </div>
            <!-- FOOTER COPYRIGHT-->
            <div id="copyright" class="row  justify-content-center align-self-bottom d-flex pb-3 ">© 2023 Copyright HUBioData Analysis Lab. </div>
        </footer>
    </div>


</body>

</html>